contracts/samples/callback/TokenCallbackHandler.sol

```diff
@@ -52,7 +41,9 @@ contract TokenCallbackHandler is IERC777Recipient, IERC721Receiver, IERC1155Rece
         return IERC1155Receiver.onERC1155BatchReceived.selector;
     }
 
-    function supportsInterface(bytes4 interfaceId) external view virtual override returns (bool) {
```

Comment:
And this seems like a line wrapping commit, maybe because the author was using 120 characters and you 80 (not sure)